backend/authapp/utils.py

```diff
@@ -0,0 +1,15 @@
+def can_modify(request, instance):
```

Comment:
What is this code is checking if the user is an administrator or not? What is preventing a user from modifying their own account privileges?

Reply:
Is this because the function only appears to be used in classes where `isAdmin` is already there?